src/seedu/addressbook/data/AddressBook.java

```diff
@@ -1,9 +1,6 @@
 package seedu.addressbook.data;
 
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Avoid auto-formatting unrelated parts of the code. Doing so can result in unrelated changes being included in the commit. You can use Sourcetree to look through changes before committing, and discard any unrelated changes.